package.json

```diff
@@ -17,6 +17,7 @@
     "pg": "^8.11.5",
     "type-graphql": "^1.2.0-rc.1",
     "typeorm": "^0.3.20",
+    "web3": "^4.9.0",
```

Comment:
Why do you need to bring web3 when you can use ethers?

Reply:
I couldn't find a bytesToHex function in the ethers documentation.
I was doing that manually at first but thought it would be better to use a library.

Reply:
does "hexlify" work?

https://docs.ethers.org/v4/api-utils.html#hex-strings

Reply:
Yup! Removing web3